### DISCUSSION TITLE
Frequent disconnection using code-server

### DISCUSSION THREAD

**Discussion by leaf2006** [coder/code-server#3158] [General]
I often disconnect the link when using the chrome browser, but this does not happen in Safari

like this:

![image](https://user-images.githubusercontent.com/51502183/115130091-1b74c580-a01f-11eb-9ec5-19dd416fc722.png)

I don’t know why it’s so strange, I hope someone can answer my question

**Comment by nooobcoder** [comment]
Can you elaborate this with logs (of browser console and code-server logs)?  @leaf2006
